mce-capi-webhook-config/deployments.cert-manaager/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mce-capi-webhook-config
+  labels:
+    app: mce-capi-webhook-config
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mce-capi-webhook-config
+  template:
+    metadata:
+      labels:
+        app: mce-capi-webhook-config
+    spec:
+      containers:
+      - args:
+          - --webhook-port=9443
+        image: quay.io/mveber/mce-capi-webhook-config
```

Comment:
The image url should be configurable. better to add env variable in the make file pass it.